### PR TITLE
[SEO] New sitemap.xml

### PR DESCRIPTION
**Pull request by j5891m**
For SEO. Linked by robot.txt. Please help to check the content.

### CODE REVIEW

File: core/sitemap.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"> 
+  <url>
+    <loc>https://freedomhkg.net/#home</loc>
```
Comment (mingchuno) [inline core/sitemap.xml]:
```suggestion
    <loc>https://freedomhkg.net/</loc>
```
???

**Comment by mingchuno** [comment]
@j5891m hang on
https://www.gatsbyjs.org/docs/creating-a-sitemap/

File: core/sitemap.xml (hunk 1 above)
Comment (j5891m) [reply core/sitemap.xml]: I suggest we can add further section links e.g. How can you help? but seems there is no anchor link to there...

**Comment by mingchuno** [comment]
close due to #36

**Pull request closed without merging**